bot/exts/halloween/hacktoberstats.py

```diff
@@ -228,45 +227,45 @@ async def get_october_prs(github_username: str) -> Optional[List[dict]]:
             f"+created:{date_range}"
             f"&per_page={per_page}"
         )
-        logging.debug(f"GitHub query URL generated: {query_url}")
+        log.logProcesses.debug(f"GitHub query URL generated: {query_url}")
 
```

Comment:
Why are you using `log.logProcesses` here? This function doesn’t appear to even be documented, I assume this is a typo.

Reply:
Yeah it was autocomplete

Reply:
Resolved in https://github.com/python-discord/sir-lancebot/pull/718/commits/c051abf49ce5bb82d127c8ddfad7ece431bb028f.